src/raft.c

```diff
@@ -399,10 +399,22 @@ static int process_REVOKE (
 		/* assume that our ack got lost */
 		rv = send_msg(OP_ACK, tk, sender, msg);
 	} else if (tk->leader != sender) {
-		tk_log_error("%s wants to revoke ticket, "
-				"but it is not granted there (ignoring)",
-				site_string(sender));
-		return -1;
+		if (tk->mode != TICKET_MODE_MANUAL) {
```

Comment:
I guess that this test would be repeating often, maybe to add a macro is_manual(tk), it would help with readability.

Reply:
ok